### DISCUSSION TITLE
[Feature]  Create photo collage based on memories

### DISCUSSION THREAD

**Discussion by achuet** [immich-app/immich#5229] [Feature Request]
### The feature

Thanks for this wonderful app. 

I think it would be great if Immich had the feature to create collages automatically or manually, similar to the feature in Google Photos. This would be a valuable addition for users who want to share their memories in a more creative and visually appealing way.

Thank you once again for your dedication to delivering this amazing open-source application.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by fabioteixei** [comment]
Yes, i think this would be the game changer that will make many people migrate from google photos to Immich.
The greatest feature of google photos is the capability to generate dynamic memories, things like "Best of Person A and Person B" or "Christmas at a Glance". 
The main reason that my wife keeps asking me to get back to google photos is to have that feature. I dont know if there is any AI algoritm out there that can do that like google does, but that would be the best feature ever.
This will probably will touch the server, since it's mostly an AI that you create this collage.

**Comment by godsgood33** [comment]
Another interesting use case that could be fun for some people.  I have 3 daughters I have added their birthdates to their pictures, what if I were to put a search or filter like "picture of child 1, child 2, and child 3 at age 4" and it would find pictures of them based on their age and offer to put them in a "smart album".  Then I could make a collage from that album.

**Comment by Batwam** [comment]
I definitely miss this feature. I wonder how hard it would be to implement something like that.

**Comment by petersuba** [comment]
Sounds like similar to #2836 - unless I am missing something?

**Reply by godsgood33** [reply]
Looks similar except for this feature is asking for it on Web and Mobile.  #2836 asks for it on the server...not sure which is most appropriate

**Reply by petersuba** [reply]
I would think the platform should be irrelevant in terms of the generation of the content - once it is generated, the presentation can happen on either

**Comment by MrScripter85** [comment]
+1 from me. I am currently migrating from Google to Immich and those collages as well as the "highlight videos" (the ones that are slowed down, 14 seconds long and have some music) are awesome. This would be the only reason to still keep my images with Google, but I hope that one day, Immich can do the same. I would be even happy to just have a function to create collages based on time/date/location/people.

**Comment by Zappes** [comment]
It might sound funny, but not being able to create those collages is exactly the reason why my wife refuses to migrate. It might not sound like something important for many, but for the people who want it, it's much more relevant than most of us could imagine.

**Comment by LybaNL** [comment]
Absolutely +1

**Comment by CnapxoK** [comment]
Hi! Has anyone already found a way to create collages?

**Comment by nickalleyne** [comment]
Just for reference, this is a duplicate of #16976 

If you want this feature, go there and give it an upvote.

**Reply by MrScripter85** [reply]
How THIS ticket here can be a duplicate if it existed two years before the ticket you mentioned?

**Reply by nickalleyne** [reply]
Good question, I am just mentioning it here because I saw this was closed and there was no link to the current one, to help others find it.

Maybe the maintainers can reply with the reason, but regardless, one is open and one is closed, so makes sense to inform people of the current one!
